resources/less/theme-berlin/light/attachments.less

```diff
@@ -13,4 +13,8 @@
             background-color: var(--grey-2);
         }
     }
+
+    .csl-bib-body a {
+        color: var(--blue-0)
```

Comment:
Do the rules look good in all five themes in light & dark?

Reply:
Subjectively, I think they look fine in all five themes :)

Reply:
Perfect, thank you!